x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/InitializePolicyContextStep.java

```diff
@@ -11,13 +11,14 @@
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.index.Index;
 
-public class InitializePolicyContextStep extends Step {
+public class InitializePolicyContextStep extends ClusterStateActionStep {
```

Comment:
As this step is special and we rely on it definitely running as is I think wee should make it final. 

Actually I think we should make all the steps final but that can be in a different change. I think its most important that this one is final as we need to ensure the creation date is definitely set properly